docs/admin/service-accounts-admin.md

```diff
@@ -8,14 +8,14 @@ title: Managing Service Accounts
 ---
 
 *This is a Cluster Administrator guide to service accounts.  It assumes knowledge of
-the [User Guide to Service Accounts](/docs/user-guide/service-accounts).*
+the [User Guide to Service Accounts](/docs/tasks/configure-pod-container/configure-service-account/).*
 
 *Support for authorization and user accounts is planned but incomplete.  Sometimes
 incomplete features are referred to in order to better describe service accounts.*
 
 ## User accounts vs service accounts
 
-Kubernetes distinguished between the concept of a user account and a service accounts
+Kubernetes distinguished between the concept of a user account and a service account
```

Comment:
suggested edit: "Kubernetes `distinguishes` between the concept of a user account and a service account for a number of reasons:"